webui/react/playwright.config.ts

```diff
@@ -6,6 +6,14 @@ import { defineConfig, devices } from '@playwright/test';
 import * as dotenv from 'dotenv';
 dotenv.config();
 
+let port: number;
+const serverAddess = process.env.PW_SERVER_ADDRESS
+if (serverAddess === undefined) {
+  throw new Error('Expected PW_SERVER_ADDRESS to be set.')
+} else {
+  port = Number(new URL(serverAddess).port)
```

Comment:
this looks good, but we might want to check if `port` exist. `Number()` returns `NaN` if input isnt number. For example, `https://localhost` doesnt work. also, i dont think else clause is needed. we can directly assign value with `const` instead of `let`